showcase/src/app/home/_home.component.scss

```diff
@@ -0,0 +1,86 @@
+h1 {
```

Comment:
We should have common behaviour for h1, h2, h3 tags so I don't think they should be set in this specific page else we will have inconsistent font sizes through the app.
Those are already defined in stark-ui/assets/styles/_typography.scss